File: stars-app/backend/models/hedis.py
from pydantic import BaseModel
from .common import Status


class HedisMeasure(BaseModel):
    measure_code: str
    measure_name: str
    weight: str
    current_rate: float
    open_gap_count: int
    status: Status
    target_rate: float
    projected_rate: float
    part: str = "C"

File: stars-app/frontend/src/types/hedis.ts
import type { Status } from "./common";

export interface HedisMeasure {
  measure_code: string;
  measure_name: string;
  weight: string;
  current_rate: number;
  open_gap_count: number;
  status: Status;
  target_rate: number;
  projected_rate: number;
  part: string;
}
